### DISCUSSION TITLE
Mix QSelect with QTree

### DISCUSSION THREAD

**Discussion by Brice155** [quasarframework/quasar#15113] [Ideas / Proposals] [answered]
Hi Quasar Team !
First, i want to congralute for the awesome work on this great framework !!!
For my needs, it would be great to mix the QSelect and the QTree components to have a behavior like the Vue Treeselect lib.
Links below : 
- Vue Treeselectin Vue 2 : https://vue-treeselect.js.org/#more-features
- Vue Treeselect in Vue 3 : https://www.npmjs.com/package/vue3-treeselect

I have a big list and i want to neste it and i didn't found a solution with the QSelect component. But maybe it exists...
Big thank !
Brayce

**Comment by metalsadman** [comment]
@Brice155 a POC https://codepen.io/metalsadman/pen/JjZgmZz

**Reply by Brice155** [reply]
Hi Metalsadam !
I'm sorry, i've just seen you're answer now...
It seems to be what i need. I'll try soon.
Thanks a lot !

**Comment by Brice155** [comment]
Hi Aldrin !
Thanks a lot.
But my needs are different. I wrote a codepen to show you the two use cases
:

   1. QSelect in menu behavior and nested lines
   2. QSelect in dialog behavior and nested lines

For both cases, i need to close by default the nested categories and so,
when i click on the arrow, it has to expand (like the QTree)

Code pen example : https://codepen.io/Brayce/pen/vYrooYp
So i didn't find a solution for this but maybe I didn't search enough ?

Any ideas would be appreciated !

Thanks a lot !!!


Le jeu. 15 déc. 2022 à 21:50, Aldrin Marquez ***@***.***> a
écrit :

> @Brice155 <https://github.com/Brice155> a POC
> https://codepen.io/metalsadman/pen/JjZgmZz
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/quasarframework/quasar/discussions/15113#discussioncomment-4413464>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFMIR3PD2WJ5YVVVVRKTUULWNOAB3ANCNFSM6AAAAAAS7SWF74>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> com>
>


-- 
Cordialement,
Brice Lallement
0688169347

**Comment by rayinfinite** [comment]
You can try using Qtree
https://codepen.io/rayinf/pen/abRNGxE?editors=1010

**Reply by Brice155** [reply, marked as answer]
Hi Rayinfinite !
Wonderful ! It is exactly what i need !
Thanks a lot for your solution !!!

**Comment by Brice155** [comment]
Rayinfinit brought me a solution.
